.circleci/config.yml

```diff
@@ -264,6 +264,53 @@ jobs:
               make localnet-start
               ./scripts/localnet-blocks-test.sh 40 5 10 localhost
 
+  snap-edge:
+      working_directory: /home/circleci/.snap_workspace/src/github.com/cosmos/cosmos-sdk
+      machine: true
+      environment:
+        GOBIN: /home/circleci/.snap_workspace/bin
+        GOPATH: /home/circleci/.snap_workspace/
+        GOOS: linux
+        GOARCH: amd64
+        GO_VERSION: "1.11.4"
+        LC_ALL: C.UTF-8
+        LANG: C.UTF-8
+      parallelism: 1
+      steps:
+        - checkout
+        - run:
+            name: Install prerequitistes
+            command: |
+              sudo apt update -y &>/dev/null
+              sudo apt install -y build-essential openssl snapd &>/dev/null
+              sudo snap install snapcraft --edge --classic
+        - run:
+            name: prepping snap build
+            command: |
+              pushd /tmp
+              wget https://dl.google.com/go/go$GO_VERSION.linux-amd64.tar.gz
+              sudo tar -xf go$GO_VERSION.linux-amd64.tar.gz
+              sudo rm -rf /usr/local/go
+              sudo mv go /usr/local
```

Comment:
Can we use a prepared docker image instead?

Reply:
I'd love it, and I'd appreciate some assistance in creating one. I guess we have a docker hub account, don't we?

Reply:
Oh BTW - no, I tried snapcraft's official and cibuilds images. None of them worked. So my last resort was to try with a VM, and that worked.